content/altinn-studio/guides/options/_index.nb.md

```diff
@@ -24,7 +24,7 @@ Dette gjøres ved å legge til feltet optionsId som referer til hvilken option (
 }
 ```
 
-### Lagre label verdi i datamodellen
+### Lagre visningsverdi i datamodellen
 Noen ganger ønsker man å lagre den viste verdien på brukerens språk i datamodellen for enklere å kunne bruke de lagrede dataene til å lagre enkle visninger uten å være avhengig av å gjøre et nytt oppslag for å få en visningsvennlig verdi. Det kan også brukes for å huske hva brukeren faktisk har sett når han valgte i tilfelle man endrer ordlyd for en verdi og vil ha logg for hva brukeren har sett.
```

Comment:
Noen ganger kan det være nyttig å lagre den viste verdien på brukerens språk i datamodellen. Det gjør det enklere å kunne bruke de lagrede dataene til å lagre enkle visninger, uten å måtte gjøre et nytt oppslag for å få en visningsvennlig verdi. Du kan også bruke dette til å huske hva brukeren faktisk har sett når han valgte, i tilfelle du endrer ordlyd for en verdi og vil ha en logg som viser hva brukeren har sett.

Kommentar: Har skrevet om til du - unngå "man" - men synes likevel at dette avsnittet og det som kommer videre nedover her er teknisk og krever forkunnskaper. Bør det legges et annet sted i dok. enn i dette introkurset? 
Eventuelt kan tittelen på denne delen hete **Mer om kodelister – for utviklere**

Reply:
Jeg forstår faktisk ikke hva dette betyr selv.